src/utils/config.py

```diff
@@ -1,6 +1,11 @@
 from enum import Enum
-
+from dotenv import load_dotenv
 from discord import Color
+import os
```

Comment:
These imports need to be sorted (I've already explained why in some review above)
They should look like this:
```py
import os
from enum import Enum

from discord import Color

from dotenv import load_dotenv
```